functions/vpn.bash

```diff
@@ -228,6 +228,8 @@ install_tailscale() {
 ##
 setup_tailscale() {
   local preAuthKey=${preauthkey}
+  # shellcheck disable=SC2154
```

Comment:
I would move this to the top of the function because it applies to multiple areas in the function.

Reply:
ok

Reply:
Actually, this is used ~5-6 times throughout this file why not make it global for the file by placing it at the top.

Reply:
hmm, we didn't in other files. But ok here